packages/gatsby/src/schema/node-model.js

```diff
@@ -297,7 +297,7 @@ class LocalNodeModel {
       runQueryActivity.start()
     }
 
-    const { entries, totalCount } = runFastFiltersAndSort({
+    const { entries, totalCount } = await getDataStore().runQuery({
```

Comment:
Now querying through datastore - the basis for future lmdb `runQuery` implementation

Reply:
Just checking if I'm reading things correctly and understand the plan - in this PR both "redux" and lmdb have same implementation (returning `Promise.resolve(runFastFiltersAndSort(args))`). In next PR(s) lmdb datastore will use its own  (potentially falling back to same thing if something is impossible or just not implemented yet)?

Reply:
Correct